chalice/app.py

```diff
@@ -266,15 +292,42 @@ def __init__(self, body, headers=None, status_code=200):
         self.headers = headers
         self.status_code = status_code
 
-    def to_dict(self):
+    def to_dict(self, binary_types=None):
```

Comment:
Makes sense. The only other place that you could put it is in the initializer, but that is something the user uses directly so it would be weird to add a binary_types there.